src/main/java/org/broadinstitute/hellbender/tools/walkers/UnmarkDuplicates.java

```diff
@@ -17,15 +17,44 @@
 import java.util.Collections;
 import java.util.List;
 
-@DocumentedFeature
+/**
+ * Removes duplicate read tags.
+ *
+ * <p> This tool "unmarks" duplicate reads in a SAM/BAM/CRAM file. Duplicate reads are defined as originated from a
+ * single fragment of DNA and can arise during sample preparation. </p>
```

Comment:
Remove the second sentence. It describes one source of duplicates but doesn't necessarily represent all of the sources of duplicates that MarkDuplicates will flag, e.g. optical duplicates.